R/Filter.R

```diff
@@ -631,3 +631,46 @@ makeFilter(
     abs(0.5 - score)
   }
 )
+
+
+#' Filter \dQuote{ranger.permutation} trains a ranger learner with
+#' \dQuote{importance = "permutation"} and assesses the variable
+#' importance for each feature.
+#'
+#' @rdname makeFilter
+#' @name makeFilter
+makeFilter(
+  name = "ranger.permutation",
+  desc = "Variable importance based on ranger permutation importance",
+  pkg  = "ranger",
+  supported.tasks = c("classif", "regr", "surv"),
+  supported.features = c("numerics", "factors", "ordered"),
+  fun = function(task, nselect, ...) {
+    lrn.type = paste0(getTaskType(task), ".ranger")
```

Comment:
Maybe `ranger.permutation` because we have at least two?

Reply:
I'm confused. The name of the filter is ranger.permutation already. What exactly do you want me to change/ reconsider?

Reply:
My bad...